### ISSUE TITLE
Signal crashing on startup

### ISSUE THREAD

**Issue by DiagonalArg** [signalapp/Signal-iOS#5339]
A friend has been trying to get Signal to work on his:

    iPhone 8 - MQ8X2LL/A
    Running iOS 15.4
    155.51GB available
    5.36.1 installed 2 days ago.

He's repeatedly installed numerous version of Signal and they all respond by crashing when he tries to start.  I was trying to help him but have no way to get a debug log.  If you can tell me how to get more information on this, I'll happily do it next time I see him.

**Comment by jrose-signal** [comment]
Thanks for reporting this! The most useful thing would be the [OS crash reports](https://support.signal.org/hc/en-us/articles/360007318591-Debug-Logs-and-Crash-Reports#ios_debug). Sorry your friend is having trouble!

**Comment by Mad-at** [comment]
I am pretty sure that this issue is connected to this one: https://github.com/signalapp/Signal-iOS/issues/5279#issue-1155383400
Please Signal team fix! We have been waiting for five months now.

**Comment by DiagonalArg** [comment]
@Mad-at  - how do you auto-submit crash reports?

I am still waiting on this as I only see the person who is having the problem once a week, and they were sick this week.

**Comment by Mad-at** [comment]
@DiagonalArg In iOS you can enable the system to automatically submit crash reports (under data protection settings). Since the more recent versions however also Signal asks you if you want to submit a crash report. 
Yesterday a new version (Version 5.37.2) was released which supposedly fixes the crash issue, but not for me.

**Comment by Mad-at** [comment]
Yes, I understand that, and I am very thankful someone is now finally looking into it. I know not a lot of people share this view, but for some people loosing their entire database is a dramatic event.

**Comment by KatieG285** [comment]
Is this an Apple issue? Opens and crashes. I don’t want to delete another app and re download. I have so much Data stored/ saved that I can’t get to. iOS 8+. Please find the cause and fix. Thank You

**Comment by DiagonalArg** [comment]
Closing in favor of [this issue](https://github.com/signalapp/Signal-iOS/issues/5424).
